sourcegraph/cody
# Chat: Fix "Ask Cody to Explain"

## Description

- Pass the correct arguments to the `cody.action.chat` command
- Add type safety to avoid this mistake again

closes https://github.com/sourcegraph/cody/issues/2970

## Test plan

1. Try "Ask Cody to Explain"
2. Check there is no error
